trunk/SUAVE/Methods/Weights/Correlations/General_Aviation/empty.py

```diff
@@ -0,0 +1,314 @@
+        t_c_h          = vehicle.wings['horizontal_stabilizer'].thickness_to_chord
+        h_tail_exposed = vehicle.wings['horizontal_stabilizer'].areas.exposed / vehicle.wings['horizontal_stabilizer'].areas.wetted
+        l_w2h          = vehicle.wings['horizontal_stabilizer'].origin[0] + vehicle.wings['horizontal_stabilizer'].aerodynamic_center[0] - vehicle.wings['main_wing'].origin[0] - vehicle.wings['main_wing'].aerodynamic_center[0] #Need to check this is the length of the horizontal tail moment arm
+        wt_tail_horizontal = tail_horizontal(S_h, AR_h, sweep_h, q_c, taper_h, t_c_h,Nult,TOW)                
+        vehicle.wings['horizontal_stabilizer'].mass_properties.mass = wt_tail_horizontal        
+    #vertical stabilizer
+    if not vehicle.wings.has_key('vertical_stabilizer'):   
+        output_3 = Data()
+        output_3.wt_tail_vertical = 0.0
+   
+        S_v = 0.0
+        warnings.warn("There is no Vertical Tail Weight being added to the Configuration", stacklevel=1)    
+        
+    else:     
+        S_v        = vehicle.wings['vertical_stabilizer'].areas.reference
+        b_v        = vehicle.wings['vertical_stabilizer'].spans.projected
+        AR_v       = (b_v**2.)/S_v
+        taper_v    = vehicle.wings['vertical_stabilizer'].taper
+        t_c_v      = vehicle.wings['vertical_stabilizer'].thickness_to_chord
+        sweep_v    = vehicle.wings['vertical_stabilizer'].sweep
+        t_tail     = vehicle.wings['vertical_stabilizer'].t_tail  
+        output_3   = tail_vertical(S_v, AR_v, sweep_v, q_c, taper_v, t_c_v, Nult,TOW,t_tail)
+        vehicle.wings['vertical_stabilizer'].mass_properties.mass = output_3.wt_tail_vertical
+    
+    #landing gear
+    if not vehicle.has_key('landing_gear'):
+        warnings.warn('There is no Landing Gear weight being added to the vehicle', stacklevel=1)
+        wt_landing_gear = Data()
+        wt_landing_gear.main = 0.0
+        wt_landing_gear.nose = 0.0
+        
+    else: 
+        landing_gear_component = vehicle.landing_gear #landing gear previously defined
+        strut_length_main      = landing_gear_component.main.strut_length
+        strut_length_nose      = landing_gear_component.nose.strut_length
+        wt_landing_gear        = landing_gear(landing_weight, Nult, strut_length_main, strut_length_nose)
+        
+        landing_gear_component.main.mass_properties.mass = wt_landing_gear.main
+        landing_gear_component.nose.mass_properties.mass = wt_landing_gear.nose
+    
+    
+    if not vehicle.has_key('avionics'):
+        warnings.warn('There is no avionics weight being added to the vehicle; many weight correlations are dependant on this', stacklevel=1)
+        avionics          = SUAVE.Components.Energy.Peripherals.Avionics()
+        W_uav = 0.
+        
+    else:
+        avionics = vehicle.avionics
+        W_uav = avionics.mass_properties.uninstalled
+    
+    has_air_conditioner = vehicle.has_air_conditioner
```

Comment:
This is weird to have a key in the vehicle of has_air_conditioner

Reply:
+1